tests/unit/conftest.py

```diff
@@ -199,6 +200,127 @@ def invalid_zigzag_config_file(tmpdir_factory,
     return filename
 
 
+@pytest.fixture(scope='session')
+def asc_zigzag_config_file(tmpdir_factory):
+    """A config for zigzag used by the ASC team"""
+
+    filename = tmpdir_factory.mktemp('data').join('config_file.json').strpath
+    config_json = \
+        """{
+              "pytest_zigzag_env_vars": {
+                "BUILD_URL": null,
+                "BUILD_NUMBER": null,
+                "RE_JOB_ACTION": null,
+                "RE_JOB_IMAGE": null,
+                "RE_JOB_SCENARIO": null,
+                "RE_JOB_BRANCH": null,
+                "RPC_RELEASE": null,
+                "RPC_PRODUCT_RELEASE": null,
+                "OS_ARTIFACT_SHA": null,
+                "PYTHON_ARTIFACT_SHA": null,
+                "APT_ARTIFACT_SHA": null,
+                "REPO_URL": null,
+                "GIT_URL": null,
+                "JOB_NAME": null,
+                "MOLECULE_TEST_REPO": null,
+                "MOLECULE_SCENARIO_NAME": null,
+                "PATH_TO_TEST_EXEC_DIR": null,
+                "MOLECULE_GIT_COMMIT": null,
+                "GIT_COMMIT": null
+              },
+              "zigzag": {
+                "test_cycle": "{{ RPC_PRODUCT_RELEASE }}",
+                "project_id": "12345",
+                "module_hierarchy": [
+                  "{{RPC_RELEASE}}",
+                  "{{JOB_NAME}}",
+                  "{{MOLECULE_TEST_REPO}}",
+                  "{{zz_testcase_class}}"
+                ],
+                "path_to_test_exec_dir": "/molecule/default/tests",
+                "build_url": "{{ BUILD_URL }}",
+                "build_number": "{{ BUILD_NUMBER }}",
```

Comment:
BTW, I think this is the right approach. I'm just a little apprehensive about not being explicit about requiring it on the "pytest-zigzag" side of things.